lib/alexa_generator/intent.rb

```diff
@@ -3,6 +3,26 @@
 
 module AlexaGenerator
   class Intent
+
+    module IntentType
```

Comment:
Sure. I want following the `SlotType` pattern.
I also saw there might be other intent types and maybe there would be different namespaces.
https://developer.amazon.com/blogs/post/Tx2EWC85F6H422/Introducing-the-ASK-Built-in-Library-Developer-Preview-Pre-Built-Models-for-Hund

```
    "intent": {

      "name": "SearchAction<object@WeatherForecast>",

      "slots": {}

    }
```